automated-comments/go/two-fer/sharewith_signature_changed.md

```diff
@@ -0,0 +1,4 @@
+The signature of the `ShareWith` function should be `ShareWith(name string) string`.
```

Comment:
```suggestion
Fix the signature of `the ShareWith` function. The expected the signature is `ShareWith(name string) string`.
```

We reallly want to push them not to change it. I know that MUST is incorrect as they can _change_ the tests, but I don't think SHOULD in this case is _actionable_ or has enough gravitas. We can be stern here.

Reply:
```suggestion
Fix the signature of the `ShareWith` function. The expected signature is `ShareWith(name string) string`.
```